src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/LocalizableStrings.resx

```diff
@@ -171,4 +171,7 @@
   <data name="CmdFormatDescription" xml:space="preserve">
     <value>Specifies the output format type for the list packages command.</value>
   </data>
+  <data name="CmdOutputVersionDescription" xml:space="preserve">
+    <value>Specifies the version of machine-readable output with '--format json' option</value>
```

Comment:
How about including 'Requires...' phrase, similar to the examples in this .resx file?

```suggestion
    <value>Sets the machine-readable format output version. Requires the '--format json' option.</value>
```

or

```suggestion
    <value>Specifies the version of machine-readable output. Requires the '--format json' option.</value>
```

Reply:
Taken 2nd recommendation.